Components/typography/typography.css

```diff
@@ -0,0 +1,46 @@
+h1{
+    font-size: 3.5rem;
+    font-weight: 700;
+}
+
+h2{
+    font-size: 2.5rem;
+    font-weight: 700;
+}
+  
+h3{
+    font-size: 1.75rem;
+    font-weight: 700;
+}
+
+h4{
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+h5{
+    font-size: 1.2rem;
+    font-weight: 700;
+}
+
+h6{
+    font-size: 1rem;
+    font-weight: 700;
+}
+
+p{
+    font-size: 1.8rem;
+}
+
+small{
+    font-size: 1.1rem;
+    font-weight: 700;
+}
+
+.text-gray{
+    color: var(--grey) !important ;
```

Comment:
avoid using !important

Reply:
I forgot to remove it. Removing this. Thank you for pointing out.